src/vivarium_gates_nutrition_optimization_child/data/loader.py

```diff
@@ -667,6 +752,15 @@ def load_gbd_2021_rr(key: str, location: str) -> pd.DataFrame:
         # Remove neonatal relative risks
         neonatal_age_ends = data.index.get_level_values("age_end").unique()[:2]
         data.loc[data.index.get_level_values("age_end").isin(neonatal_age_ends)] = 1.0
+    if key == data_keys.WASTING.RELATIVE_RISK:
```

Comment:
No change to the RR values? Can you remind me why we needed to split MAM into two categories?

Reply:
RT still has to produce the new RR values so we're using this as a placeholder.